source/screens/caseScreens/CaseOverview.tsx

```diff
@@ -111,6 +116,7 @@ const computeCaseCardComponent = (
 
   const persons = caseItem?.persons ?? [];
   const caseId = caseItem.id;
+  const pdf = caseItem?.pdf?.B ?? "";
```

Comment:
`.B`? Pdf seems to be a `Buffer` object and needs to be converted accordingly to `base64`.